lib/urlwatch/reporters.py

```diff
@@ -96,8 +102,10 @@ def get_signature(self):
                                                       copyright=urlwatch.__copyright__),
             'Website: {url}'.format(url=urlwatch.__url__),
             'Support urlwatch development: https://github.com/sponsors/thp',
-            'watched {count} URLs in {duration} seconds'.format(count=len(self.job_states),
-                                                                duration=self.duration.seconds),
+            'reported {count} of {total} watched URLs in {duration} seconds'.format(
+                count=len(self.job_states),
+                total=self.job_count_total,
+                duration=self.duration.seconds),
```

Comment:
The "reported in X seconds" is kind of wrong, as most of the time will be spent watching (hopefully).

Maybe just leave this as-is, as I'm not sure how useful this is (apart from debugging, which is already in the logs?).

Reply:
Have removed the count, it was hard to describe what the count meant. Instead, we can just say what the process did.

I think that's clearer, or, we could just remove the sentence altogether.